source/_components/cover.mqtt.markdown

```diff
@@ -45,6 +45,8 @@ Configuration variables:
 - **optimistic** (*Optional*): Flag that defines if switch works in optimistic mode. Default is `true` if no state topic defined, else `false`.
 - **qos** (*Optional*): The maximum QoS level of the state topic. Default is `0`. Will also be used when publishing messages.
 - **retain** (*Optional*): If the published message should have the retain flag on or not. Default is `false`.
+- **set_position_topic** (*Optional*): The MQTT topic to publish position commands to. Default is `None`
```

Comment:
For `set_position_topic`, I'd say "Default is `None`" can be omitted. The target audience just needs to know it's optional.

For `set_position_template`, I'd similarly reword it so it's not mentioning `None`. Something like "Default is to write the numeric position (0-100) to the `set_position_topic`.